README.md

```diff
@@ -37,21 +37,22 @@
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS3</a></li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="#"></a></li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="#"></a></li>
   </ul>
```

Comment:
- [ ] Just as the previous reviewer said, instead of leaving the server and database pace blank, please kindly remove it 👍

Reply:
Done, fixes all the issues. Please kindly help to review and approval.